tools/ci-cdk/canary-lambda/write-cargo-toml.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+#
+
+# Generates a Cargo.toml with the given AWS SDK version for this canary
+
+import argparse
+
+
+def main():
+    args = Args()
+
+    with open("Cargo.toml", "w") as file:
+        lines = [
+            '# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.',
```

Comment:
reason to not have this just be a multiline string? the template seems static?

Reply:
Reason I opted for this was because Python doesn't appear to have a built-in `trimIndent()` like Kotlin does, so the indentation would be ugly for the multi-line string.

Reply:
ah. I would probably do:
```python
base_template = """
# copyright ...
[dependencies] 
... 
...
""".strip()

def make_cargo_toml():
   ...
```